cli/start.go

```diff
@@ -155,9 +156,6 @@ func runStart(_ *cobra.Command, _ []string) error {
 		return err
 	}
 
-	info := util.GetBuildInfo()
-	log.Infof("[build] %s @ %s (%s)", info.Tag, info.Time, info.GoVersion)
```

Comment:
That makes sense, I'm not opposed to bringing it back.

On Thu, Apr 21, 2016 at 11:11 AM marc <notifications@github.com> wrote:

> In cli/start.go
> <https://github.com/cockroachdb/cockroach/pull/6096#discussion_r60598139>:
>
> > @@ -155,9 +156,6 @@ func runStart(_ *cobra.Command, _ []string) error {
> >  		return err
> >  	}
> >
> > -	info := util.GetBuildInfo()
> > -	log.Infof("[build] %s @ %s (%s)", info.Tag, info.Time, info.GoVersion)
>
> it was kind of nice having the build info in stderr as well. Without it,
> you need to match stdout (which has no timestamp) with startup lines in
> stderr.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/6096/files/ef7138bc3fd7261ac40f84762c87f45846f41fe4#r60598139>
>
-- 

-- Tobias